dynamiq/components/evaluators/llm_evaluator.py

```diff
@@ -333,7 +407,7 @@ def is_valid_json_and_has_expected_keys(
             bool: True if valid, False otherwise.
         """
         try:
-            parsed_output = json.loads(received)
+            parsed_output = self.parse_llm_json_output(received)
```

Comment:
You are just trying to extract correct json on validation, but still return raw LLM output (potentially with some trash)

Reply:
Sorry, could you please explain what do you mean one more time? This method returns the loaded json